### PR TITLE
update docstrings on orbit.py

### LINKED ISSUE #590: Small error in Orbit docstrings
🐞 **Problem**

While working with other issues, I just discovered a small error in docstrings:

https://github.com/poliastro/poliastro/blob/b3d69eca866ce443378bea36ad9c37efcb864256/src/poliastro/twobody/orbit.py#L108-L116

These docstrings are exchanged :sweat_smile: 

💡 **Possible solutions**

Just place them into their corresponding functions. Very simple issue :smile:

### PR DESCRIPTION
**Pull request by sakethramanujam**
Fixes #590

### CODE REVIEW

File: src/poliastro/twobody/orbit.py [hunk 1]
```diff
@@ -107,12 +107,12 @@ def v(self):
...
     @property
     def a(self):
-        """Semilatus rectum. """
+        """Semimajor axis. """
         return self.p / (1 - self.ecc ** 2)
 
     @property
     def p(self):
-        """Semimajor axis. """
+        """Semilatus rectum. """
```
Comment (Juanlu001) [inline src/poliastro/twobody/orbit.py]: Trailing whitespace here

**Comment by Juanlu001** [comment]
Hi @sakethramanujam, thanks for your pull request! There is some trailing whitespace, which is the reason why the continuous integration failed. Please try fixing it, and when any step fails check out the corresponding link for further information.

**Comment by sakethramanujam** [comment]
@Juanlu001 on it!

**Comment by Juanlu001** [comment]
@sakethramanujam please read the "quality" logs to find out the next steps. I recommend running `tox -e check` before committing.

**Comment by sakethramanujam** [comment]
😄 sure @Juanlu001 👍

**Comment by sakethramanujam** [comment]
I've resolved the training white spaces and after ```tox -e ``` check on my fork I'm getting the following error.

```
ERROR: InvocationError for command '/root/repo/.tox/check/bin/isort --check-only --diff --recursive --project poliastro --section-default THIRDPARTY src setup.py' (exited with code 1)
```

Any help regarding this?

**Comment by shreyasbapat** [comment]
Hi @sakethramanujam ! 
If you read the logs, there are some isort errors in __init__.py's . @Juanlu001 doesn't `isort` always skip those files?

**Comment by shreyasbapat** [comment]
On reading this:
https://github.com/timothycrosley/isort/releases/tag/4.3.5

Have a look here: 

> __init__.py has been removed from the default ignore list. The default ignore list is now empty - with all items needing to be explicitly ignored.

We have 2 options: 
1) Add `__init__.py` to ignore list
2) Fix those `__init__.py` files too.

And I would say we should fix them. @sakethramanujam can you install the latest `isort` in your system and try to fix them?

**Comment by Juanlu001** [comment]
Fixing those `__init__.py` is not a lot of work and will make the code cleaner, so let's go for it.

**Comment by Juanlu001** [comment]
Superseded by #592.

**Comment by Juanlu001** [comment]
@sakethramanujam don't let this discourage you - please take a look at the other "good first issues" and feel free to send pull requests!

**Comment by sakethramanujam** [comment]
@Juanlu001 thanks for the encouragement. Looking for those "good first issues"

**Pull request closed without merging**